sinonjs/referee-sinon
# Switch to circle

Switch to Circle and Codecov, remove Travis and coveralls


#### How to verify - mandatory
1. Observe that Circle has run the tests, and that they pass
2. Observe that coverage has been uploaded to Codecov

#### Checklist for author
